gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/PolygonDrawingViewModel.java

```diff
@@ -203,16 +211,25 @@ private void updateDrawnPolygonFeature(ImmutableList<Point> vertices) {
   }
 
   public void onCompletePolygonButtonClick() {
+    if (vertices.size() < 3 || !getFirstVertex().equals(getLastVertex())) {
```

Comment:
Does the last vertex snap to the exact value as the first index? I assume it does, just wanted to check.

Reply:
Yes, the last vertex and first vertex are exactly same.